UPGRADE-3.x.md

```diff
@@ -1,5 +1,23 @@
 # Upgrade
 
+## Alpha/Beta Schema Update
```

Comment:
@sulu/core-developer WDYT should we add a section like this or just keep it documented in a new issue/discussion on github?